xtask/src/slintdocs.rs

```diff
@@ -289,3 +290,45 @@ fn to_kebab_case(str: &str) -> String {
     }
     String::from_utf8(result).unwrap()
 }
+
+fn extract_keys() -> Vec<&'static str> {
+    let mut keys = Vec::new();
+
+    macro_rules! collect_special_key {
+        ($($char:literal # $name:ident # $($qt:ident)|* # $($winit:ident $(($_pos:ident))?)|* # $($_xkb:ident)|*;)*) => {
+            $(
+                keys.push(stringify!($name));
```

Comment:
Anstead of an "extract_keys" function, what i've done in the other function is directly put the `write!` call within the macro. Any reason not do do it like that.

Anyway this is just a detail, i think the patch is fine like that.